app/controllers/reservations_controller.rb

```diff
@@ -4,16 +4,18 @@ class ReservationsController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    reservations = Reservation.all
+    reservations = policy_scope(Reservation).where(screening: params[:screening_id])
```

Comment:
So here, you are not showing list of all reservations, but only reservations for particular screening?